tests/BenchmarkDotNet.Tests/Portability/Cpu/CpuInfoFormatterTests.cs

```diff
@@ -0,0 +1,40 @@
+﻿#if CLASSIC || NETCOREAPP2_0
+using System.Text;
+using ApprovalTests;
+using ApprovalTests.Namers;
+using ApprovalTests.Reporters;
+using BenchmarkDotNet.Portability.Cpu;
+using BenchmarkDotNet.Tests.Mocks;
+using Xunit;
+
+namespace BenchmarkDotNet.Tests.Portability.Cpu
+{   
+    [Collection("ApprovalTests")]
+    [UseReporter(typeof(XUnit2Reporter))]
+    [UseApprovalSubdirectory("ApprovedFiles")]
+    public class CpuInfoFormatterTests
+    {
+        [Fact]
+        public void FormatTest()
```

Comment:
Another ApprovalTests "feature" we should mark this tests with `[MethodImpl(MethodImplOptions.NoInlining)]` =)
See https://ci.appveyor.com/project/dotnetfoundation/benchmarkdotnet/build/0.10.11.386-aynykfpx/tests